Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -53,6 +53,7 @@
     private TagEditText mTagEditText;    /* Chips view editor for tags */
 
     private int         mBackgroundColor;
+    private boolean     mNoteChanged;
```

Comment:
I often mistap on archive when I originally mean save. In that case use would like to be asked if I would like to cancel the op as well. So something like "You have unsaved changes? Do you want to save note before archivnig? Yes, No, Cancel"

Why do we need `isNoteChanged()` and `isContentChanged()`. Shouldn't these be same?  What behavior is different content change as compared to metadata change?